sympy/integrals/transforms.py

```diff
@@ -1293,6 +1293,11 @@ def _fourier_transform(f, x, k, a, b, name, simplify=True):
     if not F.has(Integral):
         return _simplify(F, simplify), True
 
+    integral_f = integrate(f, (x,-oo,oo))
+    if integral_f is oo or integral_f is -oo or \
```

Comment:
These can be combined: `integral_f in (oo, -oo, S.NaN)`. Maybe also `integral_f.has(Integral)` should be added.

Reply:
on adding `integral_f.has(Integral)` a previous test is failing .
```python
# NOTE: the ift comes out in pieces
    assert IFT(1/(a + 2*pi*I*x), x, posk,
            noconds=False) == (exp(-a*posk), True)
```
`AssertionError`